### PR TITLE
changed Dockerfile to use VIVO-1.11.0 release

### PR DESCRIPTION
**Pull request by ht29**
changed Dockerfile to use VIVO-1.11.0 release and also remove extra vivo folder and everything is part of VIVO folder

### CODE REVIEW

File: vivo/Dockerfile [hunk 1]
```diff
@@ -7,33 +7,41 @@ MAINTAINER Quinn Hart <qjhart@ucdavis.edu>
...
-# ENV VIVO_VERSION 1.10.0
+WORKDIR /usr/local/
 
-# RUN mkdir -v /opt/vivo
-# WORKDIR /opt/vivo
-# RUN echo "$VIVO_MD5  vivo.tar.gz" > vivo.tar.gz.md5
-# RUN wget -O vivo.tar.gz https://github.com/vivo-project/VIVO/releases/download/vivo-${VIVO_VERSION}/VIVO-${VIVO_VERSION}.tar.gz && \
-#     md5sum vivo.tar.gz.md5 && \
-#     tar -xzf vivo.tar.gz && rm vivo.tar.gz
+#RUN git clone https://github.com/vivo-project/Vitro.git Vitro -b rel-1.11.0-RC
```
Comment (awoods) [inline vivo/Dockerfile:12]: Please remove unused, commented out commands.

File: vivo/Dockerfile [hunk 2]
```diff
@@ -7,33 +7,41 @@ MAINTAINER Quinn Hart <qjhart@ucdavis.edu>
...
 
 # Set properties
 # Adjust logging, you can mount over this in docker-compose to further adjust
 COPY ./log4j.properties /usr/local/tomcat/webapps/VIVO/WEB-INF/classes
+
 WORKDIR /usr/local/VIVO/home/config
+
 COPY ./dockercompose.runtime.properties runtime.properties
 COPY ./example.applicationSetup.n3 applicationSetup.n3
+#COPY ./dockercompose.runtime.properties /usr/local/vivo/home/config/runtime.properties
```
Comment (awoods) [inline vivo/Dockerfile:41]: Please remove unused, commented out commands.

**Review by awoods — CHANGES_REQUESTED** [review]

**Review by qjhart — COMMENTED** [review]
Builds and runs for me.  I'm fine with not checking the MD5 sum, and I actually kind of like the development  clue for using #RUN git clone as alternative method.

**Comment by awoods** [comment]
Thanks for the feedback, @qjhart . I will go ahead and merge.

**Review by awoods — APPROVED** [review]

**Pull request merged**